.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,40 +1,27 @@
-<!--- Hi, and thanks for contributing! -->
-<!--- Make sure to provide a general summary of your changes in the title above. -->
-<!--- For example: "[scsynth] Fix crash when encountering cute kittens" -->
+<!-- Please see CONTRIBUTING.md for guidelines. -->
 
 Purpose and Motivation
 ----------------------
 
-<!--- Why is this change required? What problem does it solve? -->
-<!--- If it fixes an open issue, please link to it here by writing "Fixes #555". -->
+Fixes #
```

Comment:
This would be really cryptic to a first-time contributor (I thought it was a typo at first) -- I think the old comment was better.